pkg/jx/cmd/common_install.go

```diff
@@ -1578,35 +1578,46 @@ func (o *CommonOptions) installProw() error {
 		kvalues = append(kvalues, "tillerNamespace=")
 	}
 
-	err = o.retry(2, time.Second, func() (err error) {
-		return o.installChart(kube.DefaultKnativeBuildReleaseName, kube.ChartKnativeBuild, "", devNamespace, true,
-			kvalues, nil, "")
-	})
+	if useKnativePipeine {
+		values = append(values, "buildnum.enabled=false", "pipelinerunner.enabled=true")
 
-	if err != nil {
-		return errors.Wrap(err, "failed to install Knative build")
+		err = o.retry(2, time.Second, func() (err error) {
+			return o.installChart(kube.DefaultKnativeBuildPipelineReleaseName, kube.ChartKnativePipeline, "", devNamespace, true,
+				kvalues, nil, "")
```

Comment:
Let's use a fixed version for this chart instead of latest, which could break the installation.